supplementary_style_guide/glossary_terms_conventions/general_conventions/i.adoc

```diff
@@ -11,37 +11,38 @@
 
 
 [discrete]
-[[ibm-eserver-system-p]]
-==== image:images/yes.png[yes] IBM eServer System p (noun)
-*Description*: Use "IBM eServer System p" for the first reference, and "IBM System p" or "System p" for subsequent references.
+[[ibm-power]]
+==== image:images/yes.png[yes] IBM Power® (noun)
+*Description*: _IBM Power_ represents the family of IBM Power processors (`ppc64le` architecture) and its products.
 
 *Use it*: yes
 
-*Incorrect forms*: pSeries
+*Incorrect forms*: pSeries, System P, Power Systems
 
 *See also*:
 
 [discrete]
-[[ibm-s-390]]
-==== image:images/yes.png[yes] IBM S/390 (noun)
-*Description*: The _IBM S/390_ is the IBM large server (or mainframe) line of computer systems. Use the full description "IBM S/390".
+[[ibm-linuxone]]
+==== image:images/yes.png[yes] IBM® LinuxONE (noun)
+*Description*: _IBM® LinuxONE_ represents the IBM® LinuxONE platform (`s390x` architecture) and its
+products.
 
 *Use it*: yes
 
-*Incorrect forms*: S/390, S90
+*Incorrect forms*: LinuxONE, linuxone
 
-*See also*:
+*See also*: xref:ibm-z[IBM Z]
 
 [discrete]
 [[ibm-z]]
-==== image:images/yes.png[yes] IBM Z (noun)
-*Description*: _IBM Z_ is the new name for the "IBM z Systems" family of IBM mainframe computers.
```

Comment:
The following index is still correct?

![Screenshot from 2023-07-12 12-53-11](https://github.com/redhat-documentation/supplementary-style-guide/assets/57954076/181d9c32-9686-40c0-8b1e-17a85d9d487c)